obspy/geodetics/tests/test_util_geodetics.py

```diff
@@ -110,6 +109,72 @@ def test_calcVincentyInverse2(self):
         self.assertAlmostEqual(alpha12, calc_alpha12)
         self.assertAlmostEqual(alpha21, calc_alpha21)
 
+    def test_calcVincentyInverseTabulated(self):
+        """ Tabulated results for Vincenty Inverse
+
+        Table II of Vincenty's paper (T. Vincenty 1975, "Direct
```

Comment:
I think this might be wrapped a little too eagerly. Wrapping should occur after 79 characters.